src/iuse_actor.h

```diff
@@ -123,10 +123,8 @@ class explosion_iuse : public iuse_actor
         // Ignored if its power field is < 0
         explosion_data explosion;
 
-        /** Maximum percentage of count that should be dropped within area of effect */
```

Comment:
They were merged into `explosion_data` earlier in the development. They aren't accessed anywhere.